common/src/View/MapFrame.cpp

```diff
@@ -177,9 +171,6 @@ namespace TrenchBroom {
 
             m_document->setViewEffectsService(nullptr);
             m_document.reset();
-
-            delete m_contextManager;
-            m_contextManager = nullptr;
```

Comment:
Deleting the context manager makes OpenGL calls (~Shader vis ~ShaderManager, possibly ~FontManager but I didn't check); it's now less visible than before so maybe add a "// FIXME: m_contextManager dsetruction makes GL calls" just so we don't forget. The order also may have changed relative to before.

I'm sort of on the fence about whether this is actually a step backwards, things with global side effects in their destructors probably shouldn't use smart pointers.

Reply:
Added a FIXME. The order should be ok still, I remember checking it.